app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -35,7 +36,7 @@
 
     final private String SEARCH = "org.fossasia.openevent.searchText";
 
-    SessionsListAdapter sessionsListAdapter;
+    TrackSessionsListAdpater trackSessionsListAdapter;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/triplez/open-event-android/file/2406654599/issues/source?bid=3233850&fileBranchId=3233912#l39)